cypress/support/commands.js

```diff
@@ -89,7 +87,5 @@ Cypress.Commands.add('createScenario', (scenarioName, isMaster, datasetOrMasterN
 
 // Delete scenario
 Cypress.Commands.add('deleteScenario', (scenarioName) => {
```

Comment:
We can probably remove the `deleteScenario` command from this file now and use `ScenarioManager.deleteScenario` instead